grafana-plugins/umh-factoryinput-panel/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "umh-factoryinput-panel",
+  "version": "1.0.0",
+  "description": "This plugins allows sending of MQTT messages to the UMH stack via the factoryinput API",
+  "scripts": {
+    "build": "grafana-toolkit plugin:build",
+    "test": "grafana-toolkit plugin:test",
+    "dev": "grafana-toolkit plugin:dev",
+    "watch": "grafana-toolkit plugin:dev --watch"
+  },
+  "author": "UMH Systems GMBH",
```

Comment:
UMH Systems GmbH

Reply:
Fixed in 00312cd29567595bd2e9c2ecb10783b3b35cfb11